talosctl-cni-bundle-install/pkg.yaml

```diff
@@ -1,6 +1,8 @@
 name: talosctl-cni-bundle-install
-variant: alpine
+variant: scratch
+shell: /bin/bash
 dependencies:
+  - stage: base
   - stage: talosctl-cni-bundle
 steps:
   - install:
```

Comment:
from / to / seems to add too much stuff into the final image

Reply:
do we need base dependency here?